### ISSUE TITLE
USB 3.0 compatibility

### ISSUE THREAD

**Issue by kozuch** [aguinet/usbtop#19]
Does this utility work for USB 3.0 devices? It does not for me at the moment - on Ubuntu 15.04 (or 15.10 - not sure now), but I am not sure about the usbtop version I am using. I know this OS is old. I will update things and try again. Just wanted to check the USB 3.0 compatibility.

**Comment by aguinet** [comment]
AFAIK, we are using the usbmon module, and don't distinguished between versions of USB. Do you still have the problem after upgrading?

**Comment by lucvdv** [comment]
It works on USB3 for me, but the speed it indicates is too low.
I copied a 16GB SD card to a disk file using dd on Ubuntu 18.04.2.
It made the copy in about 15 minutes, while usbtop showed a speed fluctuating between 9500 - 10000 kb/s.

A quick calculation in bc says that 16 GB in 15 minutes should be 18641 kB/s, almost twice as high, and with a capital B for Bytes instead of a lowercase b for bits. Unless I'm making a calculation error.

**Comment by jhihn** [comment]
I also think the USB 3 speeds are wrong.
And I think that the the "kb/s" should be "kB/s".

**Comment by Nable80** [comment]
> And I think that the the "kb/s" should be "kB/s".

Someone should finish this trivial fix: https://github.com/aguinet/usbtop/pull/26

**Comment by aguinet** [comment]
Squashed and merged for https://github.com/aguinet/usbtop/pull/26 .

**Comment by aguinet** [comment]
About the speed, I've seen various reports saying this might not be as accurate as it should be. The computation is happening here: https://github.com/aguinet/usbtop/blob/master/src/usb_stats.cpp . If you have a better way to do this, please comment here!

**Comment by crabsmaps** [comment]
I've had similar issue.
I was using aravis library with my usb3 camera and usbtop's reported speed was 16 times less than expected speed. Through some tedious debugging I've narrowed origin of incorrect calculation in libpcap. After reporting this [issue](https://github.com/the-tcpdump-group/libpcap/issues/808) it was fixed quite swiftly and I can say compiling usbtop with upstream libpcap delivers correct readings for my device.
Hope that will help somebody.
